lib/ja_serializer/serializer.ex

```diff
@@ -212,19 +212,23 @@ defmodule JaSerializer.Serializer do
 
   defp define_default_type(module) do
     type_from_module = module
-                        |> Atom.to_string
-                        |> String.split(".")
+                        |> Module.split()
                         |> List.last
                         |> String.replace("Serializer", "")
                         |> String.replace("View", "")
-                        |> JaSerializer.Formatter.Utils.format_type
+                        |> JaSerializer.Formatter.Utils.format_type()
+                        |> pluralize_type(Application.get_env(:ja_serializer, :pluralize_types))
```

Comment:
we could debate on if the pluralization should happen before or after the formatting